com.unity.ml-agents/Runtime/Demonstrations/DemonstrationStore.cs

```diff
@@ -93,7 +68,7 @@ void WriteBrainParameters(string brainName, BrainParameters brainParameters)
         /// <summary>
         /// Write AgentInfo experience to file.
         /// </summary>
-        public void Record(AgentInfo info, List<ISensor> sensors)
+        internal void Record(AgentInfo info, List<ISensor> sensors)
```

Comment:
Why not have only `public void Record(ExperienceInfo expInfo)`

Reply:
No public methods for this now.